distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1243,6 +1244,11 @@ type Column
     duplicate_count : Column
     duplicate_count self = Column_Data self.java_column.duplicateCount
 
+    ## PRIVATE
+       Helper for the expression to tell it which functions needs a Vector.
+    var_args_functions : Vector
+    var_args_functions = ['coalesce', 'min', 'max', 'sum', 'mean']
```

Comment:
Shouldnt't these be in sync with the DB Column?

Reply:
Yes was using as a test to make sure I got the right one :)